lib/changes_are_logged.rb

```diff
@@ -22,16 +22,26 @@ def log_it
         @changes_logged = {}
         save_change_log
       elsif self.changed? || !@change_comments.blank?
-        @changes_logged = self.changes
+        exclude_list = self.class.log_changes_options.fetch(:exclude, []).map(&:to_s)
+        include_list = self.class.columns.map(&:name) - exclude_list
+
+        @changes_logged = self.changes.each_with_object({}) do |(attribute, value), ret|
+          ret[attribute] = if include_list.include?(attribute.to_s)
+            value
+          else
+            [nil, 'Attribute changed, but value has been excluded.']
```

Comment:
I would expect this condition not to return a value at all — maybe there should be a `:mask` option that indicates a change was made, but the values aren't revealed in the log?